packages/core/src/tracing/index.js

```diff
@@ -179,6 +180,13 @@ exports.preInit = function preInit(preliminaryConfig) {
  * @param {CollectorPIDStore} _processIdentityProvider
  */
 exports.init = function init(_config, downstreamConnection, _processIdentityProvider) {
+  if (process.env.INSTANA_DEBUG || process.env.INSTANA_LOG_LEVEL === 'debug') {
```

Comment:
I like this check a lot 👍 

S: It would be cool if you could raise a second PR to add a utility for that. We use this check in many places e.g.

https://github.com/search?q=repo%3Ainstana%2Fnodejs%20%22if%20(process.env.INSTANA_DEBUG%20%7C%7C%20process.env.INSTANA_LOG_LEVEL%22&type=code

Reply:
Yeah sure! 👍